datasets/woz_azure/dataset.json

```diff
@@ -0,0 +1,30 @@
+{
+  "type": "dataset",
+  "id": "wozAzure",
+  "path": "woz",
+  "title": "woz",
+  "status": "beschikbaar",
```

Comment:
Is het een probleem om hier `niet_beschikbaar` van te maken. Wij hebben onlangs onze code aangepast om met deze `status` rekening te houden. Er wordt dan geen DSO API opgetrokken voor de dataset.

Reply:
prima (ben ik vergeten)